templates/trivia_qa/rc/templates.yaml

```diff
@@ -0,0 +1,99 @@
+    reference: 'Generate multiple valid answer from question with an instruction. '
+    task_template: true
+  2725b229-1217-43ed-98b0-32cf5c4293b3: !Template
+    id: 2725b229-1217-43ed-98b0-32cf5c4293b3
+    jinja: "{% if answer.type == \"WikipediaEntity\" %} \nAccording to wikipedia,\
+      \ {{question}} ||| {{answer.aliases|choice}} \n{% endif %}"
+    name: wikipedia_contextual_question
+    reference: Ask question using wikipedia keyword
+    task_template: true
+  5946db1a-a068-4a31-a06f-74a7d976cb6d: !Template
+    id: 5946db1a-a068-4a31-a06f-74a7d976cb6d
+    jinja: Guess a question from the topic "{{answer.aliases|choice}} " ||| {{question}}
+    name: guess_question
+    reference: Guess a question. It will show if model can evaluate entity in question.
+    task_template: false
+  69b1d0d7-ee22-4313-a8a0-89b718177d2a: !Template
+    id: 69b1d0d7-ee22-4313-a8a0-89b718177d2a
+    jinja: "{% if answer.type == \"WikipediaEntity\" %} \nI''ve always wondered: {{question}}{{\"\
+      \\n\"}}\nCan you search wikipedia and answer? ||| {{answer.aliases|choice}}\
+      \ \n{% endif %}\n"
+    name: context_self_description_wikipedia
+    reference: Ask question by self description from wikipedia
+    task_template: true
+  91d9f950-a25a-4557-a16f-952d74629584: !Template
+    id: 91d9f950-a25a-4557-a16f-952d74629584
+    jinja: 'Answer the following question. {{"\n"}} {{question}} ||| {{answer.aliases|choice}} '
+    name: question_with_instruction
+    reference: Question followed by an instruction
+    task_template: true
+  986dfa5a-dca4-4fe4-b895-6ce05bf49cb9: !Template
+    id: 986dfa5a-dca4-4fe4-b895-6ce05bf49cb9
+    jinja: '{{question}} {{"\n"}} Generate possible answers for the question. Answers
+      will be separated by "<SEP\>". ||| {%- for v in answer.aliases -%} {{v}} {{-
+      " <SEP\> " if not loop.last else "" -}} {%- endfor -%}'
+    name: question_multiple_valid_answer
+    reference: 'Generate multiple valid answer from question. '
+    task_template: true
+  9f2b31e8-cbf4-4c2d-8666-b31ba740e01d: !Template
+    id: 9f2b31e8-cbf4-4c2d-8666-b31ba740e01d
+    jinja: "{% if answer.type == \"WikipediaEntity\" %} \nAccording to wikipedia,\
+      \ {{question}} {{\"\\n\"}} You can try to answer multiple times. ||| {% for\
+      \ v in answer.aliases -%} Answer: {{v}} {% endfor -%}\n{% endif %}"
+    name: wikipedia_contextual_question_multiple_answer
+    reference: Ask question using wikipedia keyword. Multiple answer.
+    task_template: true
+  ab0a59db-f3f8-4111-988d-235ab27f85da: !Template
+    id: ab0a59db-f3f8-4111-988d-235ab27f85da
+    jinja: "{% if answer.type == \"WikipediaEntity\" %} \nI''ve always wondered: {{question}}{{\"\
+      \\n\"}}\nCan you answer from Wikipedia? ||| {{answer.aliases|choice}} \n{% endif\
+      \ %}\n"
+    name: context_self_description_wikipedia_2
+    reference: Ask question by self description from wikipedia
+    task_template: true
```

Comment:
hmm quite confused about this one
the model won't be able to technically search Wikipedia.
You should transform this into `Here's what I found on the internet:... please answer the question`

Reply:
same for `context_self_description_wikipedia_2`

Reply:
same for `wikipedia_contextual_question`

Reply:
same for `wikipedia_question`